### PR TITLE
imported index.js to index.html and 

### PR DESCRIPTION
**Pull request by albert-axi**
added an alert that says JS is loaded

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,9 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <script src="index.js"></script>
```
Comment (apalada-uni) [inline index.html:8]: Add a defer

**Review by apalada-uni — COMMENTED** [review]
Add a defer in index.html line 8

File: index.html (hunk 1 above)
Comment (albert-axi) [reply index.html:8]: Okay

**Comment by albert-axi** [comment]
Merge new-branch to main

**Pull request merged**